NuGet.config

```diff
@@ -2,6 +2,7 @@
 <configuration>
   <packageSources>
     <clear />
+    <add key="general-testing" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/general-testing/nuget/v3/index.json" NugetSecurityAnalysisWarningLevel="None" />
     <add key="arcade" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
```

Comment:
NugetSecurityAnalysisWarningLevel="None" is when you trigger pipepline not here. Please remove this

Reply:
<add key="general-testing" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/general-testing/nuget/v3/index.json"> add this also in  eng/common/internal/Nuget.client

Reply:
![image](https://user-images.githubusercontent.com/47157394/181640692-24d122c2-27a5-4aa0-9f41-3359281307fa.png)

this is here